javascript/rest-sveltekit/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "rest-sveltekit",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "@fontsource/fira-mono": "^4.5.10",
+    "@neoconfetti/svelte": "^1.0.0",
```

Comment:
Are these two dependencies used in the example?

Reply:
No, they are not used actually 😅
They were used in the SvelteKit demo app when I created the project with `npm create svelte@latest`
Thanks for the notice @ruheni 👍😄